Resources/Prototypes/Catalog/Research/technologies.yml

```diff
@@ -213,27 +213,217 @@
+  unlockedRecipes:
+    - MagazineRifleHighVelocity
+    - MagazinePistolHighVelocity
+
+- type: technology
+  name: technologies-non-lethal
+  id: NonLethalTechnology
+  description: technologies-non-lethal-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/disabler.rsi
+    state: base
+  requiredPoints: 5000
+  requiredTechnologies:
+    - BasicSecurityTechnology
+  unlockedRecipes:
+    - Disabler
+    - Flashbang
+    - Flash
+    - FlashPayload
+    - StunBaton
+    - Bola
+    - MagazineRifleRubber
+    - MagazinePistolRubber
+    - ShotgunShellBeanbag
+    - ShellTranquilizer
+
+- type: technology
+  name: technologies-energy-weaponry
+  id: EnergyWeaponryTechnology
+  description: technologies-energy-weaponry-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/laser_gun.rsi
+    state: icon
+  requiredPoints: 15000
+  requiredTechnologies:
+    - NonLethalTechnology
+    - CompactPowerTechnology
+  unlockedRecipes:
+    - WeaponLaserCarbine
+
+- type: technology
+  name: technologies-advanced-energy-weaponry
+  id: AdvancedEnergyWeaponryTechnology
+  description: technologies-advanced-energy-weaponry-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/advancedlasergun.rsi
+    state: icon
+  requiredPoints: 25000
+  requiredTechnologies:
+    - EnergyWeaponryTechnology
+  unlockedRecipes:
+    - WeaponAdvancedLaser
+
+- type: technology
+  name: technologies-high-power-energy-weaponry
+  id: HighPowerEnergyWeaponryTechnology
+  description: technologies-high-power-energy-weaponry-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/laser_cannon.rsi
+    state: icon
+  requiredPoints: 25000
+  requiredTechnologies:
+    - EnergyWeaponryTechnology
+  unlockedRecipes:
+    - WeaponLaserCannon
+
+- type: technology
+  name: technologies-experimental-energy-weaponry
+  id: ExperimentalEnergyWeaponryTechnology
+  description: technologies-experimental-energy-weaponry-description
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/xray.rsi
+    state: icon
+  requiredPoints: 25000
+  requiredTechnologies:
+    - EnergyWeaponryTechnology
+  unlockedRecipes:
+    - WeaponXrayCannon
+
+- type: technology
+  name: technologies-basic-armor
+  id: BasicArmorTechnology
+  description: technologies-basic-armor-description
+  icon:
+    sprite: Clothing/OuterClothing/Vests/kevlar.rsi
+    state: icon
+  requiredPoints: 10000
+  requiredTechnologies:
+    - NonLethalTechnology
+  unlockedrecipes:
+    - ClothingOuterVestKevlar
+    - ClothingHeadHelmetHelmet
+    - ClothingHeadHelmetOld
+
+- type: technology
+  name: technologies-riot-armor
+  id: RiotArmorTechnology
+  description: technologies-riot-armor-description
+  icon:
+    sprite: Clothing/Head/Helmets/light_riot.rsi
+    state: icon
+  requiredPoints: 10000
+  requiredTechnologies:
+    - BasicArmorTechnology
+  unlockedrecipes:
+    - ClothingHeadHelmetRiot
+    - ClothingOuterArmorRiot
+    - ClothingHandsGlovesCombat
```

Comment:
same here: most of this is bleh and not worth getting, considering you'll have almost all of it pre-stocked. Combat gloves have no place in here, too.